Kickstarter-iOS/Views/Controllers/CommentsViewController.swift

```diff
@@ -192,7 +192,7 @@ extension CommentsViewController {
   override func tableView(_: UITableView, didSelectRowAt indexPath: IndexPath) {
     guard let comment = self.dataSource.comment(at: indexPath) else { return }
 
-    self.viewModel.inputs.didSelectComment(comment)
+    self.viewModel.inputs.viewReplies(for: comment)
```

Comment:
I think didSelectComment is the right name for the input. We usually follow the convention of the input being similarly named to the delegate method it is called in.